tests/test_helpers/getters_helpers.ts

```diff
@@ -72,6 +72,26 @@ export function getCellContent(
   return model.getters.getCellText({ sheetId, col, row }, model.getters.shouldShowFormulas());
 }
 
+/**
+ * Get the string representation of the content of a range of cells
+ */
+export function getCellsContent(
```

Comment:
I think this name is confusing. To me "content" refers to `cell.content` (the existing helper `getCellContent` name is as confusing to me as this one).

There's already `getRangeFormattedValues` that does what you want (I think),(or `getRangeValues`)

Reply:
It's not exactly what I want, `getRangeFormattedValues` and `getRangeValues` return an array of values, I want an array of array of values. But I can use `getRangeFormattedValues` and `@prettier-ignore`

Reply:
Ah true, it returns a matrix. In odoo, we have `getEvaluatedGrid`.
Note that I like the idea of `getCellsContent` (`getEvaluatedGrid`), just the name